app/stac_api/models.py

```diff
@@ -67,6 +67,20 @@ def float_in(flt, floats, **kwargs):
     return np.any(np.isclose(flt, floats, **kwargs))
 
 
+def validate_valid_geometry(geometry):
```

Comment:
Detail: `validate_geometry` is probably sufficient as function name, I think it's obvious that we don't want to validate for 'invalid' geometries ;-)